lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -259,6 +272,13 @@ export abstract class ClientApplication {
      * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      */
     acquireTokenPopup(request: PopupRequest): Promise<AuthenticationResult> {
+        try {
```

Comment:
Is this related to SSR?

Reply:
Yes, the `preflightBrowserEnvironmentCheck` will throw an error if the window object is not defined. We don't want people to call any APIs in a non-browser environment